protobufs/livekit_sip.proto

```diff
@@ -441,7 +441,8 @@ message SIPUri {
 }
 
 enum SIPCallStatus {
-  SIP_CALL_STARTING = 0;
-  SIP_CALL_ACTIVE = 1;
-  SIP_CALL_DISCONNECTED = 2;
+  SIP_CALL_STATUS_CALL_INCOMING = 0;
```

Comment:
kind of a nit.. these enums are pretty long.. wdyt about using abbrevations like `SIP_` or `SCS_`

Reply:
and since it could be confusing what each status means, wanna just put comments in here?